Cargo.lock

```diff
@@ -3918,10 +3868,10 @@ dependencies = [
 ]
 
 [[package]]
-name = "mach"
```

Comment:
`mach` is still in our deps tree here
https://github.com/KomodoPlatform/atomicDEX-API/blob/f8ada95e976cbebf5f680d13ec39d986a22e11d8/mm2src/adex_cli/Cargo.lock#L1831
Is there a reason for not removing it completely?

Reply:
Are you sure? Seems this `Cargo.lock` file isn't in this branch

Reply:
It's not in the mm2 workspace, I think we shouldn't do it in this PR.

Reply:
> I think we shouldn't do it in this PR.

Agreed

@rozhkovdmitrii I will leave this comment to you with the related advisory [RUSTSEC-2020-0168](https://rustsec.org/advisories/RUSTSEC-2020-0168.html) so that you can update it for cli in the future if you think it's important.